packages/botonic-plugin-contentful/src/nlp/stopwords/stopwords-ro.ts

```diff
@@ -0,0 +1,358 @@
+export const roDefaultStopWords = [
```

Comment:
please add the link to where you got them from.
Why so many? there are many more than for other locales.
if possible group related words per line and add this to avoid reformat
/* eslint-disable prettier/prettier*/

Reply:
I took them from the nltk stopwords, in order to that I simply execute this python code:

Reply:
I took them from the nltk stopwords, in order to that I simply execute this python code:
```python
import nltk
from nltk.corpus import stopwords

nltk.download('stopwords')

LANGUAGE = 'romanian'

stop_words = list(set(stopwords.words(LANGUAGE)))
print(stop_words)
```

Reply:
add in the code a comment mentioning the origin (not the code) 
and sort the words alphabetically, please